calico-enterprise/threat/configuring-webhooks.mdx

```diff
@@ -0,0 +1,29 @@
+---
+description: Get security event alerts in Slack or Jira.
+title: Configuring Security Event alerts in Slack and Jira
+---
+
+# Configuring Security Event alerts in Slack and Jira
+
+You can configure {{prodname}} webhooks to post security alerts directly to a Slack channel or to create an issue in your Jira project.
+By configuring webhooks for security alerts, you can make sure that you receive critical alerts without having to sign in to Manager UI.
+
+## Before you begin
+
+Your target application must be configured to receive data from the {{prodname}} webhook.
+
+* **Slack**. You must have a webhook URL for the Slack app that you want {{prodname}} to send alerts to.
+See [Sending messages using Incoming Webhooks](https://api.slack.com/messaging/webhooks) for more information.
+* **Jira**. You must have an API token for an Atlassian user account that has write permissions to your Jira instance.
+  See [Manage API tokens for your Atlassian account](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/) for details on how to obtain an API token.
+  You also need:
+  * Your Atlassian site URL. If you access Jira at the URL `https://<your-company>.atlassian.net/jira`, then your site URL is `<your-company>.atlassian.net`.
```

Comment:
@ctauchen, could you please change `<your-company>.atlassian.net` to be `https://<your-company>.atlassian.net`. I feel like this fits better with what people expect of a URL and is a bit more consistent with the rest of the UI. Planning to including `https://` for the field in the UI.

Reply:
Main thing is that this is consistent with what the UI needs. I think without the protocol is better, but we can discuss.